src/components/textField/TextFieldMigrator.tsx

```diff
@@ -62,3 +62,6 @@ export default forwardRef(({migrate = false, ...props}: any, ref) => {
     return <OldTextField {...props} ref={ref}/>;
   }
 });
+
+TextFieldMigrator.displayName = 'TextField';
```

Comment:
Could there be a problem when two components have the same display name (`TextField` still has its `displayName`)?

Reply:
There might.. 
but it's temporary...